### PR TITLE
Add New GitHub Action for SLO Tests Integration with SDK

### PR DESCRIPTION
**Pull request by polRk**
This update introduces a new GitHub Action designed to streamline Service Level Objective (SLO) testing across our range of SDKs. This integration aims to enhance our development and testing workflows by providing automated, consistent SLO testing capabilities directly within the CI/CD pipeline.

## Key Changes:
- ### New GitHub Action: 
  - Welcome: https://github.com/ydb-platform/ydb-slo-action 
  - Implemented a GitHub Action specifically for conducting SLO tests.
  - Automates the testing process, enhancing reliability and efficiency.
- ### Improvements Over Previous System:
  - The new action now supports execution in forked repositories, unlike the previous setup that relied on external closed infrastructure and couldn't handle report submissions from forks.
  - Operates entirely on GitHub Actions infrastructure, accessible on the basic tier, making it cost-effective and easy to reproduce.
  - External contributors can now easily run and replicate SLO tests in their environments.
- ### Missing Chaos Testing:
  - While the new implementation is robust, it currently lacks chaos testing capabilities, which are planned for future updates.
- ### Impact:
  - Ensures that all SDKs meet our SLO requirements efficiently.
  - Expands accessibility for external contributors, fostering broader collaboration.
  - Reduces manual effort and potential for human error in testing processes.
  - Contributes to improved software quality and performance assurance.

### CODE REVIEW

File: .github/workflows/slo.yml [hunk 1]
```diff
@@ -1,80 +1,73 @@
...
         with:
-          name: slo-logs
-          path: logs/
+          github_pull_request_number: ${{ github.event.inputs.github_pull_request_number }}
+          sdk_name: ${{ matrix.sdk.name }}
+
+      - name: Run SLO Tests
+        run: |
+          ./tests/slo/.bin/${{matrix.sdk.id}}_linux_amd64 create grpc://localhost:2135 /Root/testdb
+          ./tests/slo/.bin/${{matrix.sdk.id}}_linux_amd64 run grpc://localhost:2135 /Root/testdb -prom-pgw localhost:9091 -report-period 500 -time 60
```
Comment (asmyasnikov) [inline .github/workflows/slo.yml:72]:
```suggestion
          ./tests/slo/.bin/${{matrix.sdk.id}}_linux_amd64 run grpc://localhost:2135 /Root/testdb -prom-pgw localhost:9091 -report-period 500 -time 60 --read-rps 1000 --write-rps 100
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ on:
 
 jobs:
   unit:
+    if: (!contains(github.event.pull_request.labels.*.name, 'no tests'))
```
Comment (rekby) [inline .github/workflows/tests.yml:15]: and no label "no slo"

File: tests/slo/build.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+export REF=$(git symbolic-ref --short HEAD)
+
+export GOOS=linux
```
Comment (rekby) [inline tests/slo/build.sh:3]: why do you need force compile linux/macos binaries instead of default platform?

File: tests/slo/run.sh [hunk 4]
```diff
@@ -0,0 +1,31 @@
+export GOOS=darwin
```
Comment (rekby) [inline tests/slo/run.sh:1]: why run script run macos arm version instead on linux x86 (or default)?

Comment (polRk) [reply tests/slo/run.sh:1]: For testing purpose, I will remove that.

**Review by rekby — APPROVED** [review]

**Pull request merged**